src/main.cpp

```diff
@@ -3565,7 +3544,8 @@ bool CVerifyDB::VerifyDB(const CChainParams& chainparams, CCoinsView *coinsview,
             return error("VerifyDB(): *** ReadBlockFromDisk failed at %d, hash=%s", pindex->nHeight, pindex->GetBlockHash().ToString());
```

Comment:
Sure, just wondered if there was a reason for doing this change only in one place, but that seems a good reason.